pkg/controller_helper/modelsource/modelhub.go

```diff
@@ -164,3 +170,36 @@ func (p *ModelHubProvider) InjectModelLoader(template *corev1.PodTemplateSpec, i
 func spreadEnvToInitContainer(containerEnv []corev1.EnvVar, initContainer *corev1.Container) {
 	initContainer.Env = append(initContainer.Env, containerEnv...)
 }
+
+func (p *ModelHubProvider) InjectModelEnvVars(template *corev1.PodTemplateSpec) {
```

Comment:
I think we have already injected the HF token in above L115. Keep one is enough.